near-contract-standards/src/non_fungible_token/core/core_impl.rs

```diff
@@ -272,10 +274,33 @@ impl NonFungibleToken {
         self.internal_transfer_unguarded(token_id, &owner_id, receiver_id);
 
         log!("Transfer {} from {} to {}", token_id, sender_id, receiver_id);
-        if let Some(memo) = memo {
+        if let Some(memo) = memo.as_ref() {
             log!("Memo: {}", memo);
         }
 
+        //default the authorized ID to be None for the logs
```

Comment:
```suggestion
        // Default the authorized ID to be None for the logs.
```

Stylistic difference but would you mind changing comments to be like this to be consistent with the rest of the repo?